react/components/Popper.tsx

```diff
@@ -77,6 +93,51 @@ const Popper = forwardRef(function Popper(props: Props, ref) {
     }
   })
 
+  const popperModifiers = useMemo(() => {
+    const handlePopperUpdate = (data: ModifierArguments<{}>) => {
+      setPlacement(data.state.placement)
+    }
+
+    const { distance, skidding } = {
+      ...defaultOffsets,
+      ...offsetsProp,
+    }
+
+    const modifiers: Partial<PopperModifier<unknown>>[] = [
+      {
+        name: 'afterWrite',
+        enabled: true,
+        phase: 'afterWrite',
+        fn: handlePopperUpdate,
+      },
+      {
+        name: 'preventOverflow',
+        options: {
+          mainAxis: true,
+          altAxis: false,
+          ...(window && { boundary: window }),
```

Comment:
If `window` is `undefined`, what would be the desired behavior? I mean, would this work without the `boundary` property?

Reply:
Nice catch, this doesn't change anything